api/src/main/java/org/pmiops/workbench/db/dao/DataSetDao.java

```diff
@@ -14,6 +14,10 @@
   List<DbDataset> findDataSetsByCohortIdsAndWorkspaceIdAndInvalid(
       long cohortId, long workspaceId, boolean dirty);
 
+  List<DbDataset> findDbDataSetsByCohortIdsAndWorkspaceId(long cohortId, long workspaceId);
```

Comment:
does this work?  I would expect it to require a singular name here.  We should make it singular regardless.

Reply:
I made singular name method and all test starts to fail since dataset itself has cohortIds and conceptids

Reply:
ahh ok that makes sense